### ISSUE TITLE
DBAL filter alias 

### ISSUE THREAD

**Issue by wdog** [Nayjest/Grids#111]
Hi I'm having trouble filtering an aliased table with DBAL

I've set set name of the filter  name of the column in the grid

```php
 (new FieldConfig())
          ->setName('indirizzo')
          ->addFilter(
            (new FilterConfig)
              ->setName('i.nome')
              ->setOperator(FilterConfig::OPERATOR_LIKE)
          ),
```

but gives me this error

```
AND (i.nome like :i.nome) ORDER BY ragsoc ASC  with params ["%sisto%"]:

Value for :i not found in params array. Params array key should be "i"
```

# fix
I've fixed this changing your code in rows 180/181 in file `DbalDataProvider.php`

```php
$fieldName_parameter = str_replace(".","_",$fieldName);
$this->src->andWhere("$fieldName $operator :$fieldName_parameter");
$this->src->setParameter($fieldName_parameter, $value);
```

now it works, maybe you can apply this change to future releases (sincerly I don't know how to pull a request )

hope it helps

**Comment by Nayjest** [comment]
Hi!
Thanks! Great contribution.

**Comment by Nayjest** [comment]
Released in v1.1.1
